code/modules/reagents/chemistry/machinery/reagentgrinder.dm

```diff
@@ -177,6 +177,9 @@
 		if(I.atom_storage.remove_type(/obj/item/food/grown, src, limit - length(holdingitems), TRUE, FALSE, user, inserted))
 			for(var/i in inserted)
 				holdingitems[i] = TRUE
+		if(I.atom_storage.remove_type(/obj/item/reagent_containers/honeycomb, src, limit - length(holdingitems), TRUE, FALSE, user, inserted))
+			for(var/i in inserted)
+				holdingitems[i] = TRUE
```

Comment:
This is suspicious, does `/obj/item/food/grown` stuff still work? It doesn't look like it'll be outputting the message anymore for grown items, let me know if I'm going senile.